sdk/src/types/block/output/native_token.rs

```diff
@@ -66,6 +70,12 @@ impl NativeToken {
     }
 }
 
+impl WorkScore for NativeToken {
```

Comment:
If in NativeTokenFeature you don't forward to this, then this is useless

Reply:
good catch

Reply:
Fixed